### ISSUE TITLE
the type name of Construct props must be in the form `${ConstructName}Props`.

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#100]
### Feature Description

Add rule when type name of props (the third argument) specified in Construct does not appear in `${ConstructName}Props`, a warning is raised. 

### Motivation

Uniform naming of Construct props enhances code readability

### Invalid Code Examples

```typescript
interface ArgsForMyConstruct {
  readonly bucket?: IBucket
}

export class MyConstruct extends Construct {
  // ❌ props name must `${ConstructName}Props` format
  constructor(scope: Construct, id: string, props: ArgsForMyConstruct) { }
}
```

### Valid Code Examples

```typescript
interface MyConstructProps {
  readonly bucket?: IBucket
}

export class MyConstruct extends Construct {
  constructor(scope: Construct, id: string, props: MyConstructProps) { }
}
```

### Rule Type

problem (possible errors)

### Implementation Details

- [ ] This rule is related to AWS CDK best practices
- [x] I'm willing to submit a PR for this feature
